lara-typescript/src/plugin-api/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@concord-consortium/lara-plugin-api",
+  "version": "3.0.0-pre.12",
+  "description": "LARA Plugin API",
+  "types": "./index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/concord-consortium/lara.git"
+  },
+  "author": "Concord Consortium",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/concord-consortium/lara/issues"
+  },
+  "homepage": "https://github.com/concord-consortium/lara#readme"
```

Comment:
It is probably better to point this at the plugin-api readme instead of the LARA readme. This confused Doug when he was trying to use the package. So in the 'old' code he changed it to: `https://github.com/concord-consortium/lara-plugin-api#readme` . It will be a different in this new code.

Reply:
Sure, I'll fix that. 
Readme is correct here:
https://www.npmjs.com/package/@concord-consortium/lara-plugin-api 
NPM uses README.md file that's actually published to NPM, not this link.